src/pad.ts

```diff
@@ -9,6 +9,8 @@ import ns from './ns'
 import { Namespace, NamedNode, st } from 'rdflib'
 import { newThing, errorMessageBlock } from './widgets'
 import { beep } from './utils'
+export { notepadToHTML } from './pad-utils'
+export { renderPartipants, participationObject, manageParticipation, recordParticipation } from './participation'
```

Comment:
This is important wrt backward-compatibility - I agree that we can have this until we do a breaking change version of solid-ui. Might want to add a comment wrt to this.